pypeit/spectrographs/spectrograph.py

```diff
@@ -187,15 +189,38 @@ def default_pypeit_par(cls):
         par['rdx']['spectrograph'] = cls.name
         return par
 
-    def config_specific_par(self, scifile, inp_par=None):
+    def config_specific_par(
+            self,
+            inp:str|list|pathlib.Path|astropy.io.fits.Header|astropy.table.Table,
+            inp_par:parset.ParSet=None
+        ):
         """
         Modify the PypeIt parameters to hard-wired values used for
         specific instrument configurations.
 
+        This method may be called with either a row from the metadata table
+        (usually from the PypeIt Reduction File) or a filename/Header (used
```

Comment:
Does this mean the type of `inp` can also be `astropy.table.row.Row`?

Reply:
Oops, it's actually a single-row table.  I missed that when cleaning up the docstrings.